src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -589,6 +1042,32 @@ public static void Test1(int x)
         }
 
         [Fact()]
+        public void AnonymousTypeSymbols_Error_InUsingStatement_OperationTree()
+        {
+            string source = @"
+class ClassA
+{
+    public static void Test1(int x)
+    {
+        using (var v1 = /*<bind>*/new { }/*</bind>*/)
```

Comment:
Consider `<bind>` around entire declaration rather than around anonymous type expression since the expression is not invalid.